functions/Get-DbaLastBackup.ps1

```diff
@@ -66,14 +67,22 @@ function Get-DbaLastBackup {
         [Alias("ServerInstance", "SqlServer")]
         [DbaInstanceParameter[]]$SqlInstance,
         [Alias("Credential")]
-        [PSCredential]$SqlCredential,
+        [PSCredential]
+        $SqlCredential,
         [Alias("Databases")]
         [object[]]$Database,
         [object[]]$ExcludeDatabase,
-        [switch][Alias('Silent')]
-        $EnableException
+        [switch][Alias('Silent')]$EnableException
     )
-    
+    begin {
+        function Get-DbaDateOrNull ($TimeSpan) {
```

Comment:
maybe Get-DbaTimeSpanOrNull. everything else checks out

Reply:
renamed that